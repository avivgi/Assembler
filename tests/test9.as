LIST:   .data   6, -9, 3
MAIN:           mov     r3, LIST[2]