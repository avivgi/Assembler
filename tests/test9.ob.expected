4 3
0100 ****!%*
0101 ***#%**
0102 **#%%*%
0103 *****%*
0104 *****#%
0105 !!!!!#!
0106 ******!
